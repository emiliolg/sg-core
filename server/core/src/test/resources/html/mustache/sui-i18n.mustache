<!DOCTYPE html>
<html sg-view="tekgenesis.service.html.Views" sg-messages="quotes : tekgenesis.samples.service.FamousQuotes" sg-params="favoriteQuote : tekgenesis.samples.service.FamousQuotes">
<head>
    <title>i18n examples</title>
</head>
<body>

<h2>All Quotes:</h2>
<div>Plato: {{quotes.PLATO}}</div>
<div>Jobs: {{quotes.JOBS}}</div>
<div>Dalai: {{quotes.DALAI}}</div>


<h2>Favorite Quote:</h2>
<div>{{favoriteQuote}}</div>

<h2>Enie y acentos:</h2>
<p>ñ y áéíóú</p>

</body>
</html>
