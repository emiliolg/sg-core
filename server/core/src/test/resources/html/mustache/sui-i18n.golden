<!DOCTYPE html>
<html sg-view="tekgenesis.service.html.Views" sg-messages="quotes : tekgenesis.samples.service.FamousQuotes" sg-params="favoriteQuote : tekgenesis.samples.service.FamousQuotes">
<head>
    <title>i18n examples</title>
</head>
<body>

<h2>All Quotes:</h2>
<div>Plato: Life must be lived as play.</div>
<div>Jobs: Stay hungry, stay foolish.</div>
<div>Dalai: Be kind whenever possible. It is always possible.</div>

<h2>Favorite Quote:</h2>
<div>Be kind whenever possible. It is always possible.</div>

<h2>Enie y acentos:</h2>
<p>ñ y áéíóú</p>

</body>
</html>
